lib/logger.js

```diff
@@ -227,6 +236,14 @@ Logger.prototype._bufferLog = function(message) {
     }
 };
 
+Logger.prototype.addAmbientMeta = function (meta) {
```

Comment:
I think, it would be better if we call it `addProperty` or something similar since `AmbientMeta` in the name of the property may confuse people - `AmbientMeta` should be something used under the hood not on the public/client-side.

Reply:
ok, I'll rename it